extensions/sns/src/commands/config/create.rs

```diff
@@ -15,7 +15,7 @@ pub fn exec(_opts: CreateOpts, dfx_cache_path: &Path) -> anyhow::Result<()> {
     let sns_config_path = if let Some(config) = Config::from_current_dir()? {
         config.get_project_root().join(CONFIG_FILE_NAME)
     } else {
-        anyhow::bail!("No config file found. Please run `dfx config create` first.");
+        anyhow::bail!("Cannot find dfx configuration file in the current working directory. Did you forget to create one?");
```

Comment:
There's an opportunity to DRY here: see this and the following three changes with identical messaging